less/index.less

```diff
@@ -48,12 +48,79 @@ table {
 	border-spacing: 0;
 }
 
-/* Set every element's box-sizing to border-box */
+// VARIABLES
+
+@themeColor: grey;
+
+@fontStack: 'ZCOOL QingKe HuangYou', cursive;
+
+// MIXINS
+
+.center() {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+  }
+
+  .twoColumns() {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+  }
+
+// GENERAL STYLES
+// Set every element's box-sizing to border-box
 * {
     box-sizing: border-box;
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
+	background-color: #000033;
+}
+
+h1, h2, h3 {
+	margin: 15px 0;
+}
+
+.pageContainer {
+	max-width: 800px;
+	margin: 0 auto;
+	padding: 15px;
+	background-color: #000066;
+	color: @themeColor;
+	font-family: @fontStack;
+	font-size: 125%;
 }
 
+header {
+	img {
+		margin: 15px auto;
+		width: 40%;
+	}
+
```

Comment:
There's no image in the header. Maybe you were trying to style the image in the "intro" section?